examples/dpo_demo_gemma3.ipynb

```diff
@@ -0,0 +1,849 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q kagglehub\n",
```

Comment:
We install `kagglehub` here, but use `huggingface_hub` below. Is it possible to use `kagglehub` throughout? Or are there any issues?